irohad/ametsuchi/impl/postgres_options.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ametsuchi/impl/postgres_options.hpp"
+#include <boost/algorithm/string.hpp>
+#include <regex>
+
+namespace iroha {
+  namespace ametsuchi {
+
+    expected::Result<PostgresOptions, std::string> PostgresOptions::create(
+        std::string postgres_options) {
+      // Split into param=value list (split by space)
+      std::regex pattern{R"([^ ]+)"};
```

Comment:
I think there are two options:
* easy way - make parsing only for one required db field